internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -1908,6 +1908,11 @@ func configShardedNewSchema(orgID, projectName, name string, diskSizeGB int, fir
 			use_replication_spec_per_shard = true
 		}
 
+		data "mongodbatlas_advanced_cluster" "test-replication-specs-per-shard-false" {
+			project_id = mongodbatlas_advanced_cluster.test.project_id
+			name 	     = mongodbatlas_advanced_cluster.test.name
+		}
```

Comment:
I believe you meant to define a plural data source here, I see in this CI this is erroring out as it is the singular data source asking for asymmetric cluster and not using `use_replication_spec_per_shard` (expected behaviour)